kubernetes/samples/charts/util/setup.sh

```diff
@@ -1,12 +1,36 @@
 #!/bin/bash
-# Copyright (c) 2018, 2019, Oracle Corporation and/or its affiliates.  All rights reserved.
+# Copyright (c) 2018,2019,2020 Oracle Corporation and/or its affiliates.  All rights reserved.
```

Comment:
This is incorrect.  It should be <begin year>,<space><most recent year>, so "2018, 2020"

Reply:
Resolved in latest update